cmd/kubernetes/kubernetes_app_remove.go

```diff
@@ -70,8 +70,7 @@ var kubernetesAppRemoveCmd = &cobra.Command{
 					output, _ := cmd.CombinedOutput()
 					fmt.Println("--------------- Uninstall script ---------------")
 					fmt.Println(string(output))
-					fmt.Println("--------------- Uninstall output ---------------")
-					fmt.Println(b.String())
+					fmt.Printf("Uninstall script for application %s not found", appName)
```

Comment:
This returns always the "not found" human-readable message even if the uninstall is executed correctly.
We expect, instead, something like:
```golang
fmt.Println("--------------- Uninstall output ---------------")
if strings.Contains(b.String(), "command not found") {
     fmt.Printf("Uninstall script for application %s not found", appName)
} else {
    fmt.Println(b.String())
}
```

Reply:
@alessandroargentieri sure, I have added the suggestions and will make sure to add newline with printf in future.